docs/content/users/install/ddev-installation.md

```diff
@@ -148,7 +148,7 @@ Installing and upgrading DDEV are nearly the same thing, because you're upgradin
 
         * Reboot if required. (Usually required.)
 
-        * Visit the Microsoft Store and install the *updated* "Windows Subsystem for Linux" and click "Open". It will likely prompt you for a username and password for the Ubuntu WSL2 instance it creates.
+        * Visit the Microsoft Store and install the *updated* ["Windows Subsystem for Linux"](https://apps.microsoft.com/store/detail/windows-subsystem-for-linux/9P9TQF7MRM4R) and click "Open". It will likely prompt you for a username and password for the Ubuntu WSL2 instance it creates.
```

Comment:
I don't think it's safe to link so explicitly here, as they change things all the time. Better to search... Not sure if the link works with just https://apps.microsoft.com/store/detail/windows-subsystem-for-linux, which would be OK.

Reply:
Maybe it's OK after all. But not sure we'll ever know if it changes.

Reply:
> Maybe it's OK after all. But not sure we'll ever know if it changes.

First of all, sorry for committing this when it wasn’t my place. The link seemed like an obvious win and I grabbed it without thinking.

Second, the Markdown link checker is pretty good complaining about broken links—so we should get a clue if/when the link stops working.